index.html

```diff
@@ -5,8 +5,45 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="icon" type="image/x-icon" href="/img/street-food.ico">
     <title>Document</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
 </head>
-<body >
-    
+<body class="wrapper">
+    <header>
+        <div class="container header">
+            <img class="logo" src="img/street-food.ico" alt="logo"/>
+            <span class="brand_name">street food</span>
+        </div>
+    </header>
+    <main class="content container">
+       <button class="btn" type="button">click me</button>
+       <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempora atque perspiciatis necessitatibus repellat modi asperiores similique? Tenetur quos, 
```

Comment:
I haven't found fonts applied to the main text. It would be the best if font family would be applied to the whole page, but then some font settings would be changed for specific cases

Reply:
Also didn't find bold and italic